components/ModelSelect.tsx

```diff
@@ -25,6 +25,9 @@ export const ModelSelect: FC<Props> = ({ model, onChange, isDark }) => {
       <option value="gpt-4-turbo">GPT-4-Turbo</option>
       <option value="gpt-4o">GPT-4o</option>
       <option value="gpt-4-0125-preview">GPT-4-0125-Preview</option>
+      <option value="gpt-4o-mini">GPT-40-Mini</option>
```

Comment:
Value looks ok, but label text is wrong, it should be "GPT-4o mini" NOT "GPT-40-Mini"